README.md

```diff
@@ -24,7 +24,7 @@ easier time rolling out new checks to teams. Win-win!
 ## How to use it
 
 Add this action to your workflow by using the [CI Standard
-Checks](https://github.com/Typeform/.github/blob/4ef297489a0a9795126e681a1211b650fa12e143/workflow-templates/ci-standard-checks.yml)
+Checks](https://github.com/Typeform/.github/blob/main/workflow-templates/ci-standard-checks.yml)
```

Comment:
I would use a static reference (i.e. the commit ID) instead of a dynamic one like `main` (it points to the latest commit to `main`)

You can use the latest commit in `main` for example.

Reply:
Thanks!